README.md

```diff
@@ -134,7 +134,15 @@ pip3 install pytest==6.2.2
 pip3 install bs4
 pip3 install paramiko
 pip3 install xlsxwriter
+pip3 install influxdb_client
+pip3 install allure-pytest
+
+
+
+
 ```
+Note: Add the pip3 module if any module not found error hits
```

Comment:
added in case someone misses some library and we will add more in future so error can come if module not found. i still like the idea for dicker. steps if we can add in readme